packages/connector-types/src/index.ts

```diff
@@ -1,21 +1,34 @@
-import type { Languages } from '@logto/phrases';
+import { Language } from '@logto/phrases';
+import { z } from 'zod';
 
 export enum ConnectorType {
   Email = 'Email',
   SMS = 'SMS',
   Social = 'Social',
 }
 
-export interface ConnectorMetadata {
-  id: string;
-  type: ConnectorType;
-  name: Record<Languages, string>;
-  logo: string;
-  description: Record<Languages, string>;
-  readme: string;
-  configTemplate: string;
+export enum ConnectorPlatform {
+  General = 'General',
+  Native = 'Native',
+  NA = 'NotApplied',
```

Comment:
what does NotApplied stand for?   Can it be avoided?

Reply:
I guess is for non-social connector, but then I think we can just make it "null"

Reply:
or optional